sample-apps/radiology/lib/configs/deepedit.py

```diff
@@ -43,23 +43,23 @@ def init(self, name: str, model_dir: str, conf: Dict[str, str], planner: Any, **
         self.tta_samples = None
 
         # Multilabel
-        # self.labels = {
-        #     "spleen": 1,
-        #     "right kidney": 2,
-        #     "left kidney": 3,
-        #     "liver": 6,
-        #     "stomach": 7,
-        #     "aorta": 8,
-        #     "inferior vena cava": 9,
-        #     "background": 0,
-        # }
-
-        # Single label
         self.labels = {
             "spleen": 1,
+            "right kidney": 2,
+            "left kidney": 3,
+            "liver": 6,
+            "stomach": 7,
+            "aorta": 8,
+            "inferior vena cava": 9,
             "background": 0,
         }
 
+        # Single label
```

Comment:
Are we making multi label as default..?

Reply:
Yes, I think it is the generic case. It covers both single and multilabel